pycm/pycm_class_func.py

```diff
@@ -178,7 +178,7 @@ def Q_calc(TP, TN, FP, FN):
 
 def TTPN_calc(item1, item2):
     """
-    Calculate TPR,TNR,PPV,NPV.
+    Calculate True positive rate (TPR), True negative rate (TNR), Positive predictive value (PPV), and Negative predictive value (NPV).
```

Comment:
I think it's too long, @sepandhaghighi what do you think?

Reply:
@alirezazolanvari 
Yeah, it's too long for function description, I think in this case abbreviations are a better option